client/js/jsonedit.js

```diff
@@ -1808,21 +1839,38 @@ function jeSet(text, dontFocus) {
 // Replace ###SELECT ME### in JSON string in jeStateNow by the string given in replaceBy,
 // display the results in the text area by calling jeSet, and select the replaced text by calling jeSelect.
 function jeSetAndSelect(replaceBy, insideString) {
+
+  const emptyBrackets = replaceBy && ((Array.isArray(replaceBy) && replaceBy.length==0) || (typeof replaceBy === 'object' && Object.keys(replaceBy).length === 0)); // ###SELECT ME### should be replaced by [] or {}
```

Comment:
`Array.isArray(replaceBy) && replaceBy.length==0` isn't really necessary because the object check should work the same for arrays. I don't mind keeping it for clarity but personally I would probably remove it.

~~I also don't see why you check `replaceBy` first.~~ Because of `null`.

Reply:
Fixed